src/components/Tool/Appeal/List/ContactRow/ContactRow.tsx

```diff
@@ -204,20 +214,14 @@ export const ContactRow: React.FC<Props> = ({
       );
 
       const givenDonations = appealDonations.map((donation) => {
-        const donationAmount =
-          donation?.appealAmount?.amount &&
-          donation?.appealAmount.convertedCurrency
-            ? currencyFormat(
-                donation.appealAmount.amount,
-                donation.appealAmount.convertedCurrency,
-                locale,
-              )
-            : donation?.appealAmount?.amount ||
-              currencyFormat(
-                0,
-                donation?.appealAmount?.convertedCurrency,
-                locale,
-              );
+        const amount = donation?.appealAmount?.amount;
+        const currency = donation?.appealAmount?.convertedCurrency;
+        let donationAmount = '';
+        if (amount && currency) {
```

Comment:
I didn't realize what you were trying to accomplish was so simple. I thought there was a branch where you needed to render the raw amount without calling currency format. Thanks for making the logic verbose and clear! If you wanted to simplify, you could do `const donationAmount = currencyFormat(amount && currency ? amount : 0, currency, locale);`